src/passes/LowerInt64.cpp

```diff
@@ -178,28 +178,28 @@ struct LowerInt64 : public Pass {
       replaceCurrent(ret);
     }
   }
-  void visitConst(Const *curr) override {
+  void visitConst(Const *curr) {
```

Comment:
The default for `WasmWalker` is to do nothing, so you can just drop all of the empty `visit*` methods.